src/x509.lisp

```diff
@@ -234,3 +253,15 @@ we are going to pass them to CFFI:WITH-POINTER-TO-VECTOR-DATA)"))
         if cn collect cn
         if (not cn) do
            (loop-finish)))))
+
+(defun get-not-after-time (cert)
```

Comment:
A docstrings is desirable for these new functions, saying at least about the return type - universal time.
And the same info in the index.html

Reply:
docs added